apps/vc-api/docs/tutorials/resident-card-tutorial.md

```diff
@@ -0,0 +1,520 @@
+
+#### [Resident] Present required credentials
+
+Initiate the credential exchange by POSTing to the `url` directly in Postman or by navigating to the `Vc Api Controller initiate Exchange` request in the collection.
+If using the collection request, fill in the `exchangeId` param to be the value used for the exchange Id by the Verifier.
+
+Send the request. A similar json should be returned in the response body:
+```json
+{
+    "errors": [],
+    "vpRequest": {
+        "challenge": "4e5aad7c-db24-4fd1-9eeb-26edb36917f1",
+        "query": [
+            {
+                "type": "PresentationDefinition",
+                "credentialQuery": [
+                    {
+                        "presentationDefinition": {
+                            "id": "286bc1e0-f1bd-488a-a873-8d71be3c690e",
+                            "input_descriptors": [
+                                {
+                                    "id": "permanent_resident_card",
+                                    "name": "Permanent Resident Card",
+                                    "purpose": "We can only allow permanent residents into the application"
+                                }
+                            ]
+                        }
+                    }
+                ]
+            }
+        ],
+        "interact": {
+            "service": [
+                {
+                    "type": "UnmediatedHttpPresentationService2021",
+                    "serviceEndpoint": "http://localhost:3000/vc-api/exchanges/1/0760682b-40c4-43ad-aea7-c8c8ee1c9035"
+                }
+            ]
+        }
+    }
+}
+```
+The `challenge` value and the final fragment of the `serviceEndpoint`, which is the `transaction id`, should be different.
+
+The response contains a VP Request, which is a specification defined here: https://w3c-ccg.github.io/vp-request-spec/.
+You can see that the VP Request's `query` section contains a `PresentationDefinition` query.
+This means that the holder must provide credentials which satisfy the `presentationDefinition`.
+
+Also note the `service` in the `interact` section of the VP Request.
+This is providing the location at which we can continue the credential request flow once we have met the `query` requirements.
+      
+### [Resident] Create the required presentation
+
+Then, open the `Vc Api Controller prove Presentation` request under the `vc-api/presentations/prove` folder.
+In the request body, use the following json, filled with your own values.
+The `challenge` should be value received from the VP Request in the previous step.
+
+```json
+{
+    "presentation": {
+        "@context":[
+            "https://www.w3.org/2018/credentials/v1",
+            "https://www.w3.org/2018/credentials/examples/v1"
+        ],
+        "type":[
+            "VerifiablePresentation"
+        ],
+        "verifiableCredential":[
+            "<FILL WITH VC RECEIVED FROM AUTHORITY>"
+        ],
+        "holder": "<FILL WITH RESIDENT DID>"
+    },
```

Comment:
Is this other request format than described in https://w3c-ccg.github.io/vc-api/#continue-exchange? It `proof` property is moved from presentation to options object

Reply:
Yes, this request isn't for continuing the exchange, but rather proving the presentation: https://w3c-ccg.github.io/vc-api/#prove-presentation